getimage.py

```diff
@@ -0,0 +1,48 @@
+import httplib, urllib, base64, random, string, json, pprint, cStringIO
```

Comment:
Imports should be separated with new lines and be ordered alphabetically. 
https://www.python.org/dev/peps/pep-0008/#imports